### DISCUSSION TITLE
Official OpenMSX support

### DISCUSSION THREAD

**Discussion by ewkohler** [maziac/DeZog#117] [Ideas]
I'm using DeZog for programming in Z80 Assembly targeting MSX. There is a branch of DeZog made by user S0urceror to use OpenMSX with DeZog. However, this version is very old and not actively maintained. Is there any possibility for DeZog to support OpenMSX natively? I'm sure a large MSX community would greatly benefit from this support. Thanks!

**Comment by maziac** [comment]
I would be happy to assist the integration of openMsx into dezog.
But I have not enough knowledge of the system to do it myself.

So, If anyone would like to volunteer please let me know.
Maybe with the help of @s0urceror ...

**Comment by S0urceror** [comment]
Hi ewkohler, I am still happily using my old build. :-)

What features would you like to be supported from the new build? Or do you just want to have it included in one source tree and not as a separate fork?

I am currently creating another Dezog related project. After that I intend to refresh my openMSX build again.

**Comment by tbytecreator** [comment]
I'm one of many MSX developer who loves DeZog and REALLY REALLY REALLY want an official's support for MSX!
